packages/web-components/src/image/image.template.ts

```diff
@@ -0,0 +1,23 @@
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+import type { Image } from './image.js';
+
+/**
+ * Template for the Image component
+ * @public
+ */
+export const template: ElementViewTemplate<Image> = html<Image>`
+  <template>
```

Comment:
Template elements are not necessary in cases like this - we only would need one here in the case we need to bind data to the host element.

Reply:
with that said, as noted below, some of the attributes which aren't specific to `img` should be enumerated here if we were to go this route.